pcsx2/gui/AppMain.cpp

```diff
@@ -529,10 +531,34 @@ extern bool FMVstarted;
 extern bool renderswitch;
 extern bool EnableFMV;
 
-void DoFmvSwitch()
+void DoFmvSwitch(bool on_off)
 {
-	ScopedCoreThreadPause paused_core( new SysExecEvent_SaveSinglePlugin(PluginId_GS) );
-	renderswitch = !renderswitch;
+	ScopedCoreThreadPause paused_core(new SysExecEvent_SaveSinglePlugin(PluginId_GS));
```

Comment:
While it's necessary to pause and unpause for the "switch to software" hack, I don't think it's necessary if only the aspect ratio is changing.